content/projekte/automatischer-rolladen/contents.lr

```diff
@@ -0,0 +1,27 @@
+title: automatischer Rolladen
+---
+author: L3D
+---
+body:
+
+Was viele nicht wissen: In der Toolbox gibt es an einigen Fenstern Automatischer Rolladen.
```

Comment:
Was viele nicht wissen: In der Toolbox gibt es an einigen Fenstern automatische Rollläden.